packages/design-system/src/types/Button/Button.d.ts

```diff
@@ -6,7 +6,7 @@ export type ButtonSize = 'small' | 'big';
 
 export type ButtonType = 'button' | 'submit';
 
-export type ButtonVariation = 'primary' | 'danger' | 'success' | 'transparent';
+export type ButtonVariation = 'primary' | 'success' | 'transparent';
```

Comment:
Same here, keep the `danger` variation for backwards-compatibility with typescript code